test cases/fortran/7 generated/meson.build

```diff
@@ -6,6 +6,11 @@ project('generated', 'fortran')
 conf_data = configuration_data()
 conf_data.set('ONE', 1)
 conf_data.set('TWO', 2)
+conf_data.set('THREE', 3)
+
+outfile = configure_file(
+      input : 'mod3.fpp', output : 'mod3.f90', configuration : conf_data)
+three = library('mod3', meson.current_build_dir() / 'mod3.f90')
```

Comment:
Why aren't you using `outfile` here directly instead of creating a string to the output?

Reply:
I see, you want to test that an absolute path to a source file works correctly.